mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AbstractMailboxMapperTest.java

```diff
@@ -240,7 +249,52 @@ public void findMailboxWithPathLikeShouldEscapeMailboxName() throws MailboxExcep
         MailboxPath regexPath = new MailboxPath(benwaInboxPath.getNamespace(), benwaInboxPath.getUser(), "INB?X");
         assertThat(mailboxMapper.findMailboxWithPathLike(regexPath)).isEmpty();
     }
-    
+
+    @Test
+    public void getEmptyAnnotationsWithNonStoredAnnotations() throws MailboxException {
+        mailboxMapper.save(benwaInboxMailbox);
+        
+        assertThat(mailboxMapper.getAnnotations(benwaInboxMailbox)).isEmpty();
+    }
+
+    @Test
+    public void getAnnotationsShouldRetrieveStoredAnnotations() throws MailboxException {
+        mailboxMapper.save(benwaInboxMailbox);
+        
+        mailboxMapper.setAnnotation(benwaInboxMailbox, ANNOTATIONS);
+
+        assertThat(mailboxMapper.getAnnotations(benwaInboxMailbox)).isEqualTo(ANNOTATIONS);
+    }
+
+    @Test
+    public void getAnnotationsByKeysShouldReturnStoredAnnotation() throws MailboxException {
+        mailboxMapper.save(benwaInboxMailbox);
+        mailboxMapper.setAnnotation(benwaInboxMailbox, ANNOTATIONS);
+
+        assertThat(mailboxMapper.getAnnotations(benwaInboxMailbox, Lists.newArrayList("/private/comment")))
+            .isEqualTo(Lists.newArrayList(PRIVATE_ANNOTATION));
+    }
+
+    @Test
+    public void setAnnotationsShouldDeleteIfNilAnnotation() throws MailboxException {
+        mailboxMapper.save(benwaInboxMailbox);
+        mailboxMapper.setAnnotation(benwaInboxMailbox, ANNOTATIONS);
```

Comment:
using ANNOTATIONS makes it harder to read because the reader has to know what's in this list to understand the expected result